### DISCUSSION TITLE
Upload Multiple PDF

### DISCUSSION THREAD

**Discussion by Luqmanasif1** [FlowiseAI/Flowise#876] [Q&A]
Hi! How can I build a ChatGPT application with multiple PDFs using Flowise? PDF will be dynamic.

**Comment by dennisprice** [comment]
I ihave the same question -- -can you help please @HenryHengZJ ?

**Comment by chungyau97** [comment]
Hi @Luqmanasif1 and @dennisprice,

You can upload multiple file as shown below:
API:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/d2702149-3373-41f6-84c1-1b6538584be5)
UI:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/1f2e153d-4b5f-4dfc-bbcb-13a814be0a07)

**Reply by TristanvanDoorn** [reply]
Thank you for this! However, when testing this route in postman, it replies 'You need to enable JavaScript to run this app.'
This implies it corresponds to the frontend somehow. Does this mean that the 'server' component is not deployed (on Railway)? Any help is highly appreciated!

**Comment by dennisprice** [comment]
Thanks for this @Luqmanasif1 ,  but you will have just imagine my slightly embarrassed laugh. Because that means absolutely nothing to me and I know it should. If I can't do it exclusively through the GUI, I can't do it. So will have to pass.  (My tech abilities go as far as pasting API keys :((

**Comment by Luqmanasif1** [comment]
Can you tell us how you uploaded two pdf in UI, in my flowise there is no option that I can upload multiple pdf(it's very strange:(().
When I selected another pdf, I replaced the previous pdf.

**Comment by chungyau97** [comment]
Hi @Luqmanasif1 and @dennisprice,

**The UI method**: you just have to `ctrl + click` or `hold LMB` to select the files you wanna upload.
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/58052ff9-c8ca-4e5e-aa06-84971a964e9a)

**Reply by dennisprice** [reply]
Too easy - thanks bro

**Comment by Luqmanasif1** [comment]
Thank you so much for your time and help. It means a lot to me.

**Comment by Luqmanasif1** [comment]
Hi! How are you?
How do I send multiple pdf files through the API endpoint in Python?

**Reply by chungyau97** [reply]
Hi @Luqmanasif1, replied in #918

**Comment by Luqmanasif1** [comment]
Thanks a lot.
I posted a one ore issue which is #922. Please tell me what I do now?
I ran my Flowise in a localhost.
